src/Microsoft.AspNetCore.Blazor/Json/SimpleJson/SimpleJson.cs

```diff
@@ -1234,6 +1250,13 @@ interface IJsonSerializerStrategy
         [SuppressMessage("Microsoft.Design", "CA1007:UseGenericsWhereAppropriate", Justification="Need to support .NET 2")]
         bool TrySerializeNonPrimitiveObject(object input, out object output);
         object DeserializeObject(object value, Type type);
+        void SetPropertyNamingScheme(int propertyNaming);
+    }
+
+    static class PropertyNamingSceme
```

Comment:
Once the enum is moved into this file, it should be possible to remove this class and its consts.